specification/maps/resource-manager/Microsoft.Maps/preview/2020-02-01-preview/maps-management.json

```diff
@@ -679,6 +679,250 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Maps/accounts/{accountName}/creators": {
+      "get": {
+        "tags": [
+          "Creators"
+        ],
+        "description": "Get all Creator instances for an Azure Map Account",
+        "operationId": "Creators_ListByAccount",
+        "x-ms-examples": {
+          "ListPrivateAtlasByAccount": {
+            "$ref": "./examples/ListMapsCreatorsByAccount.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null,
```

Comment:
>null [](start = 26, length = 4)

Do you need paging for these resources? (It's ok to declare paging in the swagger even if the service doesn't support it currently. This rename is also a good opportunity to add this without breaking changes if you need to.)

Reply:
Probably don't need paging. Creators are expensive resource. We don't expect more than one per account.